addons/ondevice-backgrounds/README.md

```diff
@@ -12,41 +10,16 @@ yarn add -D @storybook/addon-ondevice-backgrounds
 
 ## Configuration
 
-Create a file called `rn-addons.js` in your storybook config.
-
-Add following content to it:
-
-```js
-import '@storybook/addon-ondevice-backgrounds/register';
-```
-
-Then import `rn-addons.js` next to your `getStorybookUI` call.
+Then, add following content to `.storybook/main.js`:
 
 ```js
-import './rn-addons';
+module.exports = {
```

Comment:
Nice job on updating the documentation! This is something I forgot to do on controls and notes 😅 .